gulp/tasks/default.js

```diff
@@ -2,8 +2,7 @@
 
 var gulp = require('gulp');
 
-gulp.task('default', [
-    'lint',
-    'test',
-    'doc-build'
-]);
+gulp.task('default', function(done) {
+    gulp.series('lint', 'test', 'doc-build')
+    done();
+});
```

Comment:
1. Switching from gulp v3 to gulp v4 required using gulp.series() or gulp.parallel() instead of a list in task definition [reference here](https://stackoverflow.com/questions/50505275/gulp4-assertionerror-task-never-defined-when-calling-or-importing-tasks)
2. I had to signal function completion manually in gulp 4 so used callback function [reference here](https://stackoverflow.com/questions/36897877/gulp-error-the-following-tasks-did-not-complete-did-you-forget-to-signal-async)